projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1184,74 +1292,112 @@ public static function export_to_gdrive() {
+				</svg>
+				<div class="export-card__header-title"><?php echo esc_html( __( 'CSV File', 'jetpack' ) ); ?></div>
+			</div>
+			<div class="export-card__body">
+				<div class="export-card__body-description">
+					<?php echo esc_html( __( 'Download your response form data via CSV file.', 'jetpack' ) ); ?>
+				</div>
+				<div class="export-card__body-cta">
+					<?php
+					// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- we're literally building all this html to output it
+					echo $button_csv_html;
+					// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- we're literally building all this html to output it
+					echo wp_nonce_field( 'feedback_export', $this->export_nonce_field, false, false );
+					?>
+				</div>
+			</div>
+		</div>
+		<?php
 	}
 
 	/**
-	 * Print "Export to Google Drive" button.
+	 * Return HTML markup for the export to gdrive button.
+	 * If the user doesn't hold a Google Drive connection, it will return get_gdrive_connection_hint().
 	 */
-	public static function print_export_gdrive_button() {
-		$current_screen = get_current_screen();
-		if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'feedback_page_feedback-export' ), true ) ) {
-			return;
-		}
-
-		if ( ! current_user_can( 'export' ) ) {
-			return;
-		}
-
-		// if there aren't any feedbacks, bail out
-		if ( ! (int) wp_count_posts( 'feedback' )->publish ) {
+	public function get_gdrive_export_section() {
+		$user_connected = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || ( new Connection_Manager( 'jetpack' ) )->is_user_connected( get_current_user_id() );
+		if ( ! $user_connected ) {
 			return;
 		}
 
 		$user_id = (int) get_current_user_id();
 
 		require_once JETPACK__PLUGIN_DIR . '_inc/lib/class-jetpack-google-drive-helper.php';
 		$has_valid_connection = Jetpack_Google_Drive_Helper::has_valid_connection( $user_id );
-		// hint user to connect at 'https://wordpress.com/marketing/connections'
-
-		$nonce_name = 'feedback_export_nonce';
 
-		$attributes = array(
-			'data-nonce-name' => $nonce_name,
-		);
-
-		if ( ! $has_valid_connection ) {
-			$attributes['disabled'] = 'disabled';
+		if ( $has_valid_connection ) {
+			$button_html = get_submit_button(
+				esc_html( __( 'Export', 'jetpack' ) ),
+				'primary export-button export-gdrive',
+				'jetpack-export-feedback-gdrive',
+				false,
+				array( 'data-nonce-name' => $this->export_nonce_field )
+			);
+		} else {
+			$button_html = sprintf(
+				'<a href="https://wordpress.com/marketing/connections" class="button button-primary export-button export-gdrive" title="%s" rel="noopener noreferer" target="_blank">%s</a>',
```

Comment:
For all links to Jetpack.com support docs and wordPres.com pages, I would recommend using Jetpack redirects:
PCYsg-pY7-p2